spec/miq_ae_playbook_method_spec.rb

```diff
@@ -202,7 +205,10 @@
       end
 
       it "state finishes succesfully" do
+        miq_task.task_results = {'ansible_stats' => {'stats_1' => 'testing', 'stats_2' => true}}
```

Comment:
@lfu can you change stats_1 and stats_2 to var1 and var2 otherwise its confusing to see repeating strings